docs/devices_api.yml

```diff
@@ -39,8 +39,174 @@ responses:
     description: Invalid Request.
     schema:
       $ref: "#/definitions/Error"
+  UnauthorizedError: # 401
+    description: Unauthorized.
+    schema:
+      $ref: "#/definitions/Error"
 
 paths:
+  /artifacts:
```

Comment:
lets introduce the limit of max releases and document it here. the pagination we can add later without breaking this API.

Reply:
copy that. but I believe it should be documented.